scripts/lint-html.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
```

Comment:
Surprised you had to write this script. Can't we use something like https://github.com/htmllint/htmllint-cli ?

Reply:
The only reason I had to write this script is to strip the `<% %>` template tags from our HTML (in a cross-platform manner).